GEDCOMReader.py

```diff
@@ -102,7 +102,12 @@ def find_name_for_id(id):
     for individual in individuals:
         if individual.id == id:
             return individual.name
-
+        
+def age_over_150(indiv):
+    if indiv.age > 150:
```

Comment:
not sure if it's intentional but this could also be 
`return indiv.age > 150`

Reply:
@Slackow thanks, I updated the code.